tests/ert/unit_tests/gui/simulation/test_run_dialog.py

```diff
@@ -540,7 +540,6 @@ def test_run_dialog_fm_label_show_correct_info(
     assert fm_step_model._real == 0
 
     fm_step_label = run_dialog.findChild(QLabel, name="fm_step_label")
-    assert not fm_step_label.text()
```

Comment:
We can seemingly do `assert fm_step_label.text() == "Realization id 0 in iteration 0"` here?

Reply:
Yes. That is correct. I'll update this! 👍